### PR TITLE
Include newly discovered elections, update characters for CSV test, fix ward typo

### PR DESCRIPTION
**Pull request by nbdavies**
* Fix the `Ward !-2` typo in 2012-05-08 election
* In CSV test, update allowed chars in candidate name:
    - Add á for Juan Colás in 2015-04-07 general election (id 1824)
    - Disallow CR and LF (`\r, \n`)
* Include new elections in parser's set
* Add input and (untested) output files for newly discovered elections

### CODE REVIEW

File: parser.py [hunk 1]
```diff
@@ -433,7 +433,7 @@ def get_result_for_json(filename):
         election = json.load(jsonfile)
         get_election_result(election)
 
-
+# http://openelections.net/api/v1/election/?format=json&limit=0&state__postal=WI
```
Comment (nbdavies) [inline parser.py:436]: Including the full URL on one line for easier copy/pasting

File: tests/csv-test-config.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ fields:
     isOneOf: DEM,REP,IND,,CON,NP,LIB,NA,WIG,NA,GRE,WGR,AME
   candidate:
     - matches:
-      # Line break, letter, number ()&,".+ or spaace
-      - ^[\r\na-z0-9-()\&,"\.\+' ]+$
+      # Letter, number ()&,".+ or spaace
+      - ^[áa-z0-9-()\&,"\.\+' ]+$
```
Comment (nbdavies) [inline tests/csv-test-config.yml:27]: For Juan B. Colás, as he is known in the 2015-04-07 election. (Exactly 6 years prior on 2009-04-07, he was listed as "Juan B. Colas")

**Review by davipo — COMMENTED** [review]
Does "New input and output files" commit include changes to 2012-05-08 results to fix ward typo?

**Comment by davipo** [comment]
I think there are too many files changed here, it's hard to scroll to find anything. In future, better to commit one year at a time when adding new elections.

**Comment by davipo** [comment]
Need list of new elections added, for "Output files without tests" issue #11.

**Comment by davipo** [comment]
List of results files, extracted from "Files changed" tab on this page:

20110215__wi__primary__ward.csv
20110712__wi__primary-recall__ward.csv
20110712__wi__special__primary__ward.csv
20110719__wi__general-recall__ward.csv
20110719__wi__primary-recall__ward.csv
20110809__wi__general-recall__ward.csv
20110809__wi__special__general__ward.csv
20110816__wi__general-recall__ward.csv
20120221__wi__primary__ward.csv
20120508__wi__primary-recall__ward.csv
20130219__wi__primary__ward.csv
20131022__wi__special__primary__ward.csv
20131119__wi__special__general__ward.csv
20140218__wi__primary__ward.csv
20140401__wi__general__ward.csv
20150407__wi__general__ward.csv
20160216__wi__primary__ward.csv
20171219__wi__special__primary__ward.csv
20180116__wi__special__general__ward.csv
20180220__wi__primary__ward.csv

**Pull request merged**